web-bluetooth/read-characteristic-value-changed.js

```diff
@@ -15,8 +15,8 @@ function onReadBatteryLevelButtonClick() {
 
 function requestDevice() {
   log('Requesting Bluetooth Device...');
-  return navigator.bluetooth.requestDevice(
-    {filters: anyNamedDevice(), optionalServices: ['battery_service']})
+  return navigator.bluetooth.requestDevice({
+      acceptAllDevices: true, optionalServices: ['battery_service']})
```

Comment:
It makes me sad that we are using acceptAllDevices so liberally. I recognize the desire to make our samples super easy to use but every developer that copies this code will end up using it which will result in a bunch of unrelated devices being shown in the chooser and energy being wasted because we have no filters.